src/components/Content/Card.js

```diff
@@ -10,7 +10,16 @@ import { Footer } from '../Footer/Footer';
 export const Card = () => {
   return (
     <CardContainer>
-      <CardDetails />
+      <CardDetails>
+        <Text size={'25px'} margin={'0 0 10px 0'} color={'red'}>
```

Comment:
Maybe you can make red is a constant?

Reply:
I will work on the card component in the next pr